### DISCUSSION TITLE
Problems with Lagrangian particle visualization in ParaView

### DISCUSSION THREAD

**Discussion by wyldckat** [blueCFD/Core#190] [Q&A] [answered]
This was a question sent to us via contact form recently:

> Greetings!
> 
> This is actually a problem in many versions of paraview, but I encountered in BlueCFD 2020-1 too and I have to use this version with windows...
> 
> So, when lagrangian particles used (current example: coalChemistryFoam/simplifiedSiwek), I cant visualize the particles in paraview(initialized with paraFoam command). I am sharing the error below, it is pretty self explanatory but It is impossible to change lines in all timestep data. Is there a way around for this?
> 
> ```
> Core-2020\OpenFOAM-8\tutorials\lagrangian\coalChemistryFoam\simplifiedSiwek\0.0425/lagrangian/limestoneCloud1/positions: Expected punctuation token ')', found 0
> ```
> 
> Best regards,
> Eren.

**Comment by wyldckat** [comment, marked as answer]
For this, we developed the following function object: https://github.com/blueCFD/lagrangianExtraFunctionObjects

Usage instructions are provided in the `README.md` file and look into the respective branch for more instructions: https://github.com/blueCFD/lagrangianExtraFunctionObjects/branches

**Reply by ErenC1** [reply]
Greetings Mr. Santos!

I actually saw that link while looking for a solution, but there were no files (and there are still no files) so asked directly to you. Now I found this link, I guess it is already compiled with 2020 version, and this instructions will save me I think.
https://github.com/blueCFD/lagrangianExtraFunctionObjects/tree/blueCFD-Core-2017#how-to-getbuild-lagrangianextrafunctionobjects

Thank you for your time,
Have a great new year,
Eren.

**Comment by wyldckat** [comment]
Also see issue #111 for more examples on what can be done in these situations.
